woeip/templates/upload.html

```diff
@@ -0,0 +1,8 @@
+{% extends '__base.html' %}
```

Comment:
Nit: Move this to `woeip/apps/air_quality/templates/air_quality/upload.html`. This keeps your templates organized so that they are easy to manage as you introduce more templates and apps.

Reply:
I'd like to keep all the templates centralized in `woeip/templates`. The templates are dependent on each other's structure. It makes sense to keep them together, to make their dependencies easier to navigate. I created a 'woeip/templates/air_quality' folder instead. This can be revisited, if this organization style becomes cumbersome.